Cargo.toml

```diff
@@ -37,25 +37,25 @@ slog-async = "2.3"
 slog-scope = "4.0"
 slog-stdlog = "3.0"
 slog-term = "2.4"
-byteorder = "0.5"
+byteorder = "1.2.4"
```

Comment:
Why specify the minor version? I think `1.2` is enough.

Reply:
I use `cargo upgrade <dep>` to upgrade the crate, it simply add the newest version(include minor verions) automatically, just like `cargo add` or copy from the crates.io .  I checked there is some crate also specify the minor version before, like `lazy_static = "0.2.1"` , most others without minor version.
If there is a convention, please let me know, I will remove the minor version. thanks

Reply:
Thanks! I agree with @hicqu , generally we should avoid using minor versions.